swiper.el

```diff
@@ -503,6 +503,17 @@ When non-nil, INITIAL-INPUT is the initial search pattern."
   (interactive)
   (swiper--ivy (swiper--candidates) initial-input))
 
+;;;###autoload
+(defun swiper-region-or-not ()
+  "If region is selected, `swiper' with the keyword selected in region.
+If the region isn't selected, `swiper'."
+  (interactive)
+  (if (region-active-p)
+      (progn (setq mark-active nil)
+	     (swiper (buffer-substring
```

Comment:
Please follow the project's `dir-locals-file` settings and indent with spaces, not tabs.

Reply:
Fix indent with speces.